projects/packages/forms/src/dashboard/inbox/response.js

```diff
@@ -1,8 +1,11 @@
 import { dateI18n, getSettings as getDateSettings } from '@wordpress/date';
+import { Fragment } from '@wordpress/element';
 import { __, sprintf } from '@wordpress/i18n';
 import classnames from 'classnames';
+import { map } from 'lodash';
```

Comment:
Do we need to use `lodash` here, or can use regular native `map`?

Reply:
`lodash` is already a dependency on the project, so we try to stay consistent and use the same lib everywhere for manipulations. Vanilla/native is always an option, but since the dependency is already there...